nextjs_frontend/src/app/proposal/[cpfDevedor]/components/ProposalPodium/proposal-podium.tsx

```diff
@@ -0,0 +1,54 @@
+"use client";
+
+import { useState } from "react";
+import { Acordo } from "@/models/Acordos";
+import ProposalCard from "../ProposalCard/proposal-card";
+import Styles from "./proposal-podium.module.scss";
+
+interface ProposalPodiumProps {
+  proposals: Acordo[];
+  changeProposal: (proposal: Acordo) => void;
+}
+
+export default function ProposalPodium({
+  proposals,
+  changeProposal
+}: ProposalPodiumProps) {
+  const [selectedCardID, setSelectedCardID] = useState<number>();
```

Comment:
Setting a default selected card makes sense to me as when there is no card featured the proposals appear to be out of order (2nd -> 1st -> 3rd)